### ISSUE TITLE
How can I create a rule only for the browser and not it's WPA applications?

### ISSUE THREAD

**Issue by bigplayer-ai** [newlooper/VirtualSpace#56]
Hey,
How can I create a rule for the Edge browser but allow/exclude (not be triggered by the rules) its installed Web Platform Applications (WPA)? I have installed Microsoft Edge WPA, such as the iCloud Notes application. 
For instance, I have the iCloud notes application installed as a Microsoft Edge WPA application with the following launch command:
```
"C:\Program Files (x86)\Microsoft\Edge\Application\msedge_proxy.exe" --profile-directory=Default --app-id=gocfomdjloapejkgmbakonlbbbnjhmnp --app-url=https://www.icloud.com/notes/ --app-launch-source=4
```
With these settings, iCloud Notes are moved to another desktop because msedge.exe has a rule. Like in the image below:
![msedge.exe to the fun.](https://user-images.githubusercontent.com/119108387/223647269-b1ec02cf-d91f-4919-a2e3-1369e1b10b67.png)

However, I want to move only the Microsoft Edge browser (msedge.exe) and allow/exclude the Microsoft Edge WPA (iCloud Notes) to stay on the active desktop. 
The main difference between multiple profiles/applications on Microsoft Edge/Google Chrome is the shortcut arguments. 
The different profile/application open the same executable called msedge.exe/chrome.exe, and the shortcuts/processes only differ in their arguments.

TLDR:

> The user wants to know how to create a rule for the Edge browser that moves it to another desktop but does not affect its installed Web Platform Applications (WPA), such as iCloud Notes. 
> The user has installed iCloud Notes as a WPA using msedge_proxy.exe, which is a file that redirects the shortcut to the main msedge.exe application. 
> The problem is that msedge.exe has a rule that moves it to another desktop, and this also applies to iCloud Notes because it uses msedge.exe as well. The user wants to know how to exclude specific WPA (iCloud Notes) from this rule.

Best Regards,

**Comment by bigplayer-ai** [comment]
TLDR:
The user wants to know how to create a rule for the Edge browser that moves it to another desktop but does not affect its installed Web Platform Applications (WPA), such as iCloud Notes. 
A WPA is a website that can be installed on your device as a web application. The user has installed iCloud Notes as a WPA using msedge_proxy.exe, which is a file that redirects the shortcut to the main msedge.exe application. 
The problem is that msedge.exe has a rule that moves it to another desktop, and this also applies to iCloud Notes because it uses msedge.exe as well. The user wants to know how to exclude iCloud Notes from this rule.
